po/sv.po

```diff
@@ -40,15 +40,15 @@ msgstr "Stäng flik"
 
 #: Tab.cpp:216
 msgid "Confirm"
-msgstr ""
+msgstr "Bekräfta"
 
 #: Tab.cpp:240
 msgid "Enter"
-msgstr ""
+msgstr "Ok"
```

Comment:
Is there a better way to translate "enter" (this is a verb here, "enter [this text to the page]") than "OK"? It says "OK" in GNOME Web because their dialog is just bad, but we get to do it properly.

GNOME HIG says:
> Label the affirmative button with a specific imperative verb, for example: _Save_ or _Print_. This is clearer than a generic label like _OK_ or _Done_.

Reply:
I did check Firefox, Gnome Web, Vivaldi, the Group for Swedish Computer Terminology and searched the internet. That was the best I could come up with. ¯\_(ツ)_/¯

Reply:
Hm? You seem to have translated "enter" (imperative) as "ange" elsewhere. Perhaps that same verb has an infinitive form?